process/templates/PROJECT_NAME_graduation_stage.md

```diff
@@ -0,0 +1,36 @@
+## Project graduation application
+
+### List of project maintainers
+The project must have maintainers with a minimum of three different company affiliations.
+  * "name, affiliation, GitHub ID"
+
+### Mission of the project
+The project must be aligned with the OpenSSF mission and either be a novel approach for existing areas, address an unfulfilled need, or be code needed to deliver OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+  * "description of the project mission"
+
+### Project adoption
+The project must be able to show adoption by multiple parties, which could be production deployments or substantial use by established open source communities, and demonstrate the value of that adoption to either the end users or the open source community.
+  * "description of adoption"
+
+### Release cadence
+The project must be able to show a consistent release cadence.
+
+### Governance
+Projects must have documented project governance and be able to demonstrate that governance in action.
+  * "link to governance documents"
+
+### Security audit
+When applicable, projects must have completed a security audit through a third party and addressed audit findings and recommendations.
+  * "link to results of security audit"
```

Comment:
We should certainly make it *possible*, but do we want to *mandate* a security audit for every project to be called an "OpenSSF project"? I don't know that we have the funds for that.

Reply:
I think we can support a mandate like that; typically I'd expect that to be requested of, and then approved by, the TAC Funding Committee. We may run into a situation where either the TAC funding committee's allocation is too low to fund it, or the TAC balks at funding the whole amount because it's a large code base and thus a large audit cost, in which case we can see if the folks behind the project can fund or co-fund it with the TAC, or whatever. Worst case, projects can't move to Graduated because no one is willing to fund the audit - which I've never seen actually happen, but worth keeping in mind as a possible outcome from a policy decision like this.

Reply:
You both make good points. Here's my thinking on this:
- "When applicable" gives the TAC some discretion here, but it is likely that it would apply to most Projects that have code or a service as a primary output.
- if a project stands in contrast to other projects because it _alone_ cannot gain enough support to complete even one security audit, I would see that that as signal that the project may not have sufficient support to graduate at that time. On the other hand, if multiple projects aren't able to meet this bar, and the TAC isn't able to fund it, I would want the TAC to revisit this requirement.

Reply:
+1 @AevaOnline